examples/activities/form_activity.py

```diff
@@ -0,0 +1,56 @@
+import asyncio
+import logging.config
+from pathlib import Path
+
+from symphony.bdk.core.activity.command import CommandActivity, CommandContext
+from symphony.bdk.core.activity.form import FormReplyActivity, FormReplyContext
+from symphony.bdk.core.config.loader import BdkConfigLoader
+from symphony.bdk.core.service.message.message_service import MessageService
+from symphony.bdk.core.symphony_bdk import SymphonyBdk
+
+
+async def run():
+    async with SymphonyBdk(BdkConfigLoader.load_from_symphony_dir("config.yaml")) as bdk:
+        await bdk.activities().register(SlashGifCommandActivity(bdk.messages()))
+        await bdk.activities().register(ReplyFormReplyActivity(bdk.messages()))
+        await bdk.datafeed().start()
+
+
+class SlashGifCommandActivity(CommandActivity):
+    def __init__(self, messages: MessageService):
+        self._messages = messages
+        super().__init__()
```

Comment:
why do we call init here and not in the ReplyFormReplyActivity?

Reply:
**I will remove this along with the constructor of the `CommandActivity`**

```Python
def __init__(self):
    self.bot_display_name = str
```
This was done in java and I think it's purpose was to set the bot name in the activity.
ActivityRegistery.java : `a command activity (potentially) needs the bot display name in order to parse the message text content` (maybe we don't actually need it)